exdb/views.py

```diff
@@ -239,3 +239,30 @@ def form_valid(self, form):
             experience.save()
             return HttpResponseRedirect(self.get_success_url())
         return super(EditExperienceView, self).form_valid(form)
+
+
+class SearchExperienceResultsView(ListView):
+    access_level = 'basic'
+    context_object_name = 'experiences'
+    template_name = 'exdb/search_results.html'
+    model = Experience
+
+    def get_queryset(self):
+        token = self.request.GET.get('search')
+        if not token:
+            return Experience.objects.none()
+        token.strip()
+        Qs = Q(name__icontains=token) | Q(description__icontains=token) | Q(goal__icontains=token) |\
+            Q(guest__icontains=token) | Q(guest_office__icontains=token) | Q(conclusion__icontains=token) |\
+            Q(keywords__name__icontains=token) | Q(recognition__name__icontains=token) |\
+            Q(recognition__affiliation__name__icontains=token) |\
+            Q(planners__first_name__icontains=token) | Q(planners__last_name__icontains=token) |\
+            Q(author__first_name__icontains=token) | Q(author__last_name__icontains=token) |\
+            Q(type__name__icontains=token) | Q(sub_type__name__icontains=token)
```

Comment:
We should tokenize the search query and ensure that each word is contained in at least one column.